AWS-README.md

```diff
@@ -109,18 +109,18 @@ If WildFly is running in standalone mode:
 @AwsEc2Instance(
         nameTag = "instanceName",
         region = "region",
-        mode = EapMode.STANDALONE,
+        mode = WildFlyMode.STANDALONE,
         standalone = @StandaloneMode(
                 user = "mngmtUser",
                 password = "mngmtPassword",
                 port = "mngmtPort"))
-static OnlineManagementClient staticEapClientSpecified;
+static OnlineManagementClient client;
 ```
 
 All values may contain expressions (`${my.property}`):
 - `nameTag` - mandatory, value of `name` tag of EC2 Instance
 - `region` - optional, if empty, `sunstone.aws.region` from `sunstone.properties` will be used
-- mode - optional, default to EapMode.STANDALONE
+- mode - optional, default to WildFlyMode.STANDALONE
```

Comment:
What about `ServerMode` ? :-)

Reply:
Hmm, I have never heard of it in context of WildFly. But I have found `OperatingMode`, wdyt?